game/scripts/npc/items/farming/item_greater_travel_boots.txt

```diff
@@ -48,8 +48,7 @@
     //-------------------------------------------------------------------------------------------------------------
     "AbilityCooldown"                                     "45.0"
     "AbilitySharedCooldown"                               "travel"
-    "AbilityChannelTime"                                  "2.75 2.5 2.25 2"
-
+    "AbilityChannelTime"                                  "2"
```

Comment:
I like buffing travels, I think it's a great idea, but I don't like that it removes scaling. Boots are already hard to convince people to upgrade with the only exceptions being tranqs/arcanes for obvious reasons. Perhaps make something else about it scale nicely, or maybe just bump all the levels over by 1? `2.5 2.25 2 1.75`?